src/api/routes/index.js

```diff
@@ -22,7 +21,6 @@ router.use('/admin', adminRouter);
 router.use('/bookmarks', bookmarkRouter);
 router.use('/notifications', notificationRouter);
 
-router.use('/profiles', authenticate, profileRouter);
```

Comment:
@truestbyheart why remove this?

Reply:
@MemunaHaruna this line was added during my previous rebase and I couldn't spot it I previously removed it when I was working on my follow and unfollow task.